core/ir/instr_shared.c

```diff
@@ -3050,82 +3037,80 @@ instr_convert_to_isa_regdeps(void *drcontext, instr_t *instr_from)
                 /* Map sub-registers to their containing register.
                  */
                 reg_id_t reg_canonical = reg_to_pointer_sized(reg);
-                opnd_size_t reg_size = reg_get_size(reg_canonical);
-                uint reg_size_in_bytes = opnd_size_in_bytes(reg_size);
                 if (!dst_reg_used[reg_canonical]) {
                     ++num_dsts;
                     dst_reg_used[reg_canonical] = true;
-                    if (reg_size_in_bytes > max_reg_size)
-                        max_reg_size = reg_size_in_bytes;
                 }
             }
         }
     }
 
     /* Retrieve number of register source operands from real ISA instruction.
+     * We use max_src_opnd_size_bytes to keep track of the size of the largest source
+     * operand.
+     * We count the number of bytes instead of using opnd_size_t to avoid relying on
+     * OPSZ_ enum values.
+     * We convert max_src_opnd_size_bytes to its corresponding OPSZ_ enum value later on,
```

Comment:
nit: I would make these bullets

Reply:
Mhm, it's not really a list of "same-level things" that belong to the same "semantic category", it's more like a description of logical steps that follow one another and a brief explanation about why we're doing what we're doing.
However, the retrieval of the number of source operands and the setting of max_src_opnd_size_bytes are indeed two different things (they are together in the code for convenience), so I separated this in two comments and highlighted the sequence of "logical steps" we follow for max_src_opnd_size_bytes.